scalahost/nsc/src/main/scala/scala/meta/internal/semantic/Inferred.scala

```diff
@@ -0,0 +1,36 @@
+package scala.meta.internal.semantic
+
+// data structure to manage multiple inferred sugars at the same position.
```

Comment:
I have a feeling that conversion should be represented by a different data structure. Or maybe it doesn't need any data structure at all? After all, it has a unique range position.